guides/v2.3/magento-functional-testing-framework/2.3/getting-started.md

```diff
@@ -137,101 +140,97 @@ Example: `MAGENTO_ADMIN_USERNAME=admin`
 * `MAGENTO_ADMIN_PASSWORD` must contain the user password required for authorization in the Admin area.
 Example: `MAGENTO_ADMIN_PASSWORD=123123q`
 
-The following self-descriptive variables have included default values.
+{: .bs-callout .bs-callout-info }
+If the `MAGENTO_BASE_URL` contains a subdirectory like `http://magento.test/magento2ce`, specify `MAGENTO_CLI_COMMAND_PATH`.
 
-```config
-SELENIUM_HOST=127.0.0.1
-SELENIUM_PORT=4444
-SELENIUM_PROTOCOL=http
-SELENIUM_PATH=/wd/hub
-```
+### Step 5. Enable the Magento CLI commands
 
-{:.bs-callout .bs-callout-warning}
-Only change or specify `SELENIUM_*` values if you are not running Selenium locally, or if you have changed your Selenium Server configuration.
+In the `magento2/dev/tests/acceptance` directory, run the following command to enable the MFTF to send Magento CLI commands to your Magento instance.
 
-Your environment settings form the path to your running Selenium Server.
-Example:
-```
-http://127.0.0.1:4444/wd/hub
+ ```bash
+cp dev/tests/acceptance/.htaccess.sample dev/tests/acceptance/.htaccess
 ```
 
-### Step 5. (Optional) Copy `command.php` into Magento installation {#add-cli-commands}
+#### (Optional) Copy `command.php` into Magento installation {#add-cli-commands}
 
-If you are installing the MFTF not from your Magento installation, locate the `command.php` file in the MFTF:
+If you are installing the MFTF from the other location than your Magento installation, point to the `command.php` file in the MFTF source code:
 
 `magento2-functional-testing-framework/etc/config/command.php`
 
-And copy it into your Magento installation under: 
+And copy it into your Magento installation: 
 
-`magento2/dev/tests/acceptance/utils/command.php`
+`magento2/dev/tests/acceptance/utils/`
 
-If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project].
+If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project][].
 
 {:.bs-callout .bs-callout-warning}
 If you do not have access to your Magento installation and cannot complete the step you will not be able to use the Magento CLI commands.
 
-### Step 6. Enable the Magento CLI commands
+### Step 7. Generate and run tests   {#run-tests}
 
-In the `magento2/dev/tests/acceptance` directory, run the following command to enable the MFTF to send Magento CLI commands to your Magento instance.
+To run tests, you need a running Selenium server and [`mftf`][] with required parameters.
```

Comment:
To run tests, you need an active Selenium server and .... 

I don't understand what this is:  and [mftf] with required parameters ....

You use the mftf CLI commands to run the tests.....   should it be that you  need an active server and know how to use the xxx commands, or  something along those lines?   Or  "MFTF must be installed and configured on your local system.